### DISCUSSION TITLE
Suggestion : Webview in development mode at development time

### DISCUSSION THREAD

**Discussion by serge-hulne** [wailsapp/wails#2316] [Ideas]
It would be handy to have Webview in development (automatically, buy default) mode at development time, so that one would not have to :
- Right click on the app.
- Click on `inspect element`
- Click on `console`
every time one hits the 'save' button in the editor (triggering a Webview reboot).

**Comment by serge-hulne** [comment]
Perhaps, this is a suggestion which ought to be forwarded to the Webview for Go developers?

**Comment by stffabi** [comment]
There's a debug option for that https://wails.io/docs/reference/options#openinspectoronstartup

**Reply by serge-hulne** [reply]
Super ! Thanks!
